stripe/http_client.py

```diff
@@ -51,9 +51,14 @@
 except ImportError:
     urlfetch = None
 
+# proxy support for the pycurl client
+from urlparse import urlparse
+
 
 def new_default_http_client(*args, **kwargs):
     if urlfetch:
+        # no proxy support in urlfetch. for a patch, see:
+        # https://code.google.com/p/googleappengine/issues/detail?id=544
```

Comment:
I think we may be able to remove this note given that it's duplicated from within the URL fetch client's constructor.